docs/content/Caching/Caching.md

```diff
@@ -190,6 +124,77 @@ cube(`Orders`, {
+
+To aid with this, Cube.js defines a `refreshKey` for each cube. [Refresh keys](cube#parameters-refresh-key) are evaluated by Cube.js to assess if the data needs to be refreshed.
+
+```js
+cube(`Orders`, {
+  // This refreshKey tells Cube.js to refresh data every 5 minutes
+  refreshKey: {
+    every: `5 minute`
+  }
+
+  // With this refreshKey Cube.js will only refresh the data if
+  // the value of previous MAX(created_at) changed
+  // By default Cube.js will check this refreshKey every 10 seconds
+  refreshKey: {
+    sql: `SELECT MAX(created_at) FROM orders`
+  }
+});
+```
+
+By default, Cube.js will check and invalidate the cache in the background when in [development mode][link-development-mode]. When development mode is disabled you can set `CUBEJS_SCHEDULED_REFRESH_TIMER=true` to enable this behavior.
+
+We recommend enabling background cache invalidation in a separate Cube.js worker for production deployments. Please consult the [Production Checklist][link-production-checklist] for more information.
+
+[link-production-checklist]: /deployment/production-checklist
+[link-development-mode]: /configuration/overview#development-mode
+[link-production-checklist-refresh]: /deployment/production-checklist#set-up-refresh-worker
+
+If background refresh is disabled, Cube.js will refresh the cache during query
+execution. Since this could lead to delays in responding to end-users, we
+recommend always enabling background refresh.
+
+### Default Refresh Keys
+
+The default values for `refreshKey` are
+ * `every: '2 minute'` for BigQuery, Athena, Snowflake, and Presto.
+ * `every: '10 second'` for all other databases.
+
++You can use a custom SQL query for checking if a refresh is required by changing the [`refreshKey`](/cube#parameters-refresh-key) property in a cube's Data Schema. Often, a `MAX(updated_at_timestamp)` for OLTP data is a viable option, or examining a metadata table for whatever system is managing the data to see when it last ran.
+
+
+### How to disable the cache?
+
+There's no straightforward way to disable caching in Cube.js.
+The reason for it is Cube.js is not just stores cached values but uses the cache as a point of synchronization and coordination between nodes in a cluster.
+For the sake of design simplicity, Cube.js doesn't distinguish client invocations, and all calls to the data load API are idempotent.
+This provides excellent reliability and scalability but has some drawbacks.
+One of those load data calls can't be traced to specific clients, and as a consequence, there's no guaranteed way for a client to initiate a new data loading query or know if the current invocation wasn't initiated earlier by another client.
```

Comment:
Not sure if we need a better transition to this sentence. What is the word "those" referring to? Would it be better to say "Individual load data calls can't be...."?

Reply:
The whole section needs a rewrite here. In this PR I'm just moving it. I'd recommend leave it as it for now and then rewrite the whole piece in the future PRs.